### ISSUE TITLE
Half-Life doesn't launch on Ubuntu but Steam thinks it's playing.

### ISSUE THREAD

**Issue by RealDoigt** [ValveSoftware/halflife#3224]
When I click play, steam indicates that the game is playing, but nothing happens. I've tried waiting for a long time, I've tried uninstalling and reinstalling, I rebooting the pc and I googled the issue but no cigar. It's weird because HL2 works and this game works on crappier pcs I have at home, but not my main one. I'm on Kubuntu (KDE 5.21.4; Ubuntu 21.04), 64 bit system with 16 gb ram

I was told by steam support to open an issue here regarding this as they basically told me they can't help.

**Comment by tmp64** [comment]
Related to #3112, #3130. Try disabling shader caching in Steam settings, maybe it will help

**Comment by RealDoigt** [comment]
yep that solved it. It's unfortunate though that it's a general setting and not per game.
